app.py

```diff
@@ -796,7 +957,7 @@ def save_program (user):
         program_count = user ['program_count']
     db_set('users', {'username': user ['username'], 'program_count': program_count + 1})
 
-    return jsonify({'name': name})
+    return jsonify({})
```

Comment:
We should use the version in test, not this one.

Reply:
Fixed!